teps/0098-workflows.md

```diff
@@ -412,16 +434,236 @@ and Pipeline definitions into a higher level opinionated syntax that is aware of
 
 ## Proposal
 
+Initially, the cluster operator will install Tekton Workflows onto their Kubernetes cluster. They will connect an individual repo
+a set of repos associated with the same organization in their SCM, allowing Tekton Workflows to receive events from this repo.
+This part of the user journey is discussed in [Repo Connections](#repo-connections).
+
+The cluster operator can then define Workflows for their repo. While Workflows can be defined directly on a cluster or in a separate
+configuration repo, most users are expected to interact with Workflows by committing their Workflow definitions directly to the repo
+being operated on. Special considerations for this user journey are discussed in [In-Repo Configuration](#in-repo-configuration).
+
+The [API](#api) section goes into more detail on the proposed API, with examples.
+A Workflow definition encodes a PipelineRun and the events that trigger it, plus some built-in functionality for the most common use cases.
+This section also includes default Workflow templates that users can create with the Tekton CLI.
+
+Lastly, the [Milestones](#milestones) section proposes a roadmap for implementing the changes in this proposal.
+
+### Repo Connections
+
+The cluster operator or DevOps engineer needs an easy way to connect their repo to Tekton Workflows, so that Workflows
+can trigger PipelineRuns when events occur on the repo. Cluster operators shouldn't have to set up webhooks or apps
+themselves, or create Kubernetes secrets containing credentials; these steps will happen automatically during Workflows setup.
```

Comment:
i think there is going to be a token issue in case of webhook, the workflow would not necessary have a access to the repos and the user may need to provide them?

Reply:
Oh sorry, what I meant here was that the cluster operator shouldn't have to create a webhook secret themselves (for securing the webhook). The other credentials (e.g. token) can be in a k8s secret or entered via the CLI